### PR TITLE
Fix README - Removed the part that suggest to remove the master taint

### PR DESCRIPTION
**Pull request by vishnoianil**
suggest to remote the master taint

Signed-off-by: Anil Vishnoi <vishnoianil@gmail.com>

### CODE REVIEW

**Comment by vishnoianil** [comment]
@danwinship @dcbw PTAL, thanks

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # How to use Open Virtual Networking with Kubernetes
 
-On Linux, the easiest way to get started is to use OVN daemonset and deployments.
+On Linux, the easiest way to get started is to use OVN DaemonSet and Deployments.
 
 ## Install Open vSwitch kernel modules on all hosts.
 
-Most Linux distributions come with openvswitch kernel module by default.  You
+Most Linux distributions come with OpenvSwitch kernel module by default.  You
```
Comment (danwinship) [inline README.md:7]: well, if you're going for correctness, it's "Open vSwitch" with a space

File: README.md [hunk 2]
```diff
@@ -23,17 +23,13 @@ sudo apt-get build-dep dkms
...
-
-Create OVN daemonset and deployment yamls from templates by running the commands below:
+Create OVN StatefulSet, DaemonSet and Deployment yamls from templates by running the commands below:
 (The $MASTER_IP below is the IP address of the machine where kube-apiserver is
-running). Note, when specifying the pod CIDR to the command below, daemonset.sh will
+running). 
+
+**Note:** when specifying the pod CIDR to the command below, daemonset.sh will
 generate a /24 subnet prefix to create per-node CIDRs. Ensure your pod subnet is has a
 prefix less than 24, or edit hte  generated ovn-setup.yaml and specify a host subnet
```
Comment (danwinship) [inline README.md:34]: typo "hte" (and an extra space after it)

**Comment by danwinship** [comment]
> suggest to remote the master taint

remoVe

also, fix the commit message (and PR title) to have the whole sentence on one line

File: README.md (hunk 1 above)
Comment (vishnoianil) [reply README.md]: done

**Comment by vishnoianil** [comment]
@danwinship Addressed all the comments. PTAL, thanks

**Comment by girishmg** [comment]
@vishnoianil you changed the title to include `Statefulset` but none of the yamls that we are applying in the README is statefulset.

Note that  `ovnkube-db.yaml` is a deployment. Only `ovnkube-db-raft.yaml` is Statefulset. 

Perhaps you need to add a new section on how to run `ovnkube-db` in RAFT mode?

If you don't want to tackle it in this PR, then remove Statefulset from README and tackle RAFT and Statefulset in the next PR. Thanks.

**Comment by vishnoianil** [comment]
@girishmg Fixed. Removed the StatefulSet, we can address that in other PR.

**Pull request merged**